style.css

```diff
@@ -316,3 +304,310 @@ footer {
+  }
+
+  .decoration {
+    display: none;
+  }
+
+  .decoration-desktop {
+    display: inline;
+    margin-right: 0;
+    flex: 1;
+  }
+
+  .hero-section1 {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    margin-bottom: 100px;
+  }
+
+  .hero-section1 .container {
+    flex: 1;
+  }
+
+  .bg-image {
+    background: url(./assets/desktop/image_dots.png) no-repeat;
+    background-position: right bottom;
+  }
+
+  .paragraph {
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 21px;
+    letter-spacing: -0.01em;
+    width: 500px;
+    height: 84px;
+    margin-top: 10px;
+  }
+
+  .cta-btn {
+    width: 254px;
+    height: 60px;
+    margin-bottom: 120px;
+  }
+
+  /* Project-section */
+  .project-header {
+    position: relative;
+    background: url(./assets/desktop/image_dots.png) no-repeat top right;
+  }
+
+  .rotate-one {
+    transform-origin: right top;
+    transform: rotate(-90deg);
+    position: absolute;
+    right: 90%;
+  }
+
+  .section {
+    padding-top: 0;
+    margin-bottom: 110px;
+  }
+
+  .grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 0.5fr 1fr;
+  }
+
+  .project-item {
+    position: relative;
+  }
+
+  .container-information {
+    position: absolute;
+    width: 100%;
+  }
+
+  .project-img {
+    width: 100%;
+  }
+
+  .pose-a {
+    position: absolute;
+  }
+
+  .top {
+    top: 20%;
+  }
+
+  .bottom {
+    bottom: 0;
+  }
+
+  .top-20 {
+    top: -30%;
+  }
+
+  .bg-square {
+    background: url(./assets/desktop/Group83.png) no-repeat right 60%;
+    background-size: 85%;
+  }
+
+  .w-70 {
+    width: 70%;
+  }
+
+  .span-2 {
+    grid-column: span 2;
+  }
+
+  .span-3 {
+    grid-column: span 3;
+  }
+
+  .hide {
+    display: none;
+  }
+
+  /* About-section */
+  .container-item {
+    display: flex;
+    flex-direction: row-reverse;
+    padding-inline-start: 110px;
+    padding-inline-end: 80px;
+    margin-top: 110px;
+    margin-bottom: 100px;
+    column-gap: 80px;
+  }
+
+  .about1 {
+    font-size: 90px;
+  }
+
+  .skill-heading {
+    margin-bottom: 20px;
+    font-size: 20px;
+    font-weight: 800;
+  }
+
+  .skills-lists {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    flex: 2;
+  }
+
+  .about-content {
+    flex: 1;
+  }
+
+  .bi-project6 {
+    background-image: url(./assets/desktop/image_dots.png), url(./assets/desktop/Shapeblue.png);
+    background-position: 5px 200px, 175px 0;
+    background-repeat: no-repeat;
+    background-size: 60%, 40%;
+  }
+
+  /* Footer  */
+  .container-footer {
+    display: grid;
+    grid-template-columns: 1fr 2fr 1fr;
+    background-image: none;
+  }
+
+  .bg-footer {
+    background-color: #f9f9fa;
+  }
+
+  .right {
+    position: relative;
+  }
+
+  .right img {
+    position: absolute;
+    bottom: 0;
+  }
+
+  .footer-heading {
+    margin-top: 0;
+  }
+
+  .last-form {
+    display: grid;
+    grid-template-columns: 1fr 0.01fr 1fr;
+  }
+
+  .last-grid1 {
+    padding-right: 70px;
```

Comment:
- [ ] Kindly make sure you use the appropriate margin and spacing as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/188288442-a414326c-5ad3-4eb4-84cc-105f3291f528.png)  |  ![image](https://user-images.githubusercontent.com/74814780/188288433-2cc7281e-f2e3-4b32-96da-4fc77ade390f.png)